arctic/store/metadata_store.py

```diff
@@ -2,12 +2,12 @@
 import logging
 
 import pandas as pd
-
 import bson
 import pymongo
+
+from .._util import indent
```

Comment:
_minor_ lexicographic order

Reply:
But '_' comes before (lower case) letters. And this is consistent with VersionStore ordering.